sensor_msgs/Image img
ClassBox[] boxes
---
ClassBox[] new_boxes